app/services/hmpps_api/prison_api/offender_api.rb

```diff
@@ -29,32 +29,43 @@ def self.list(prison, page = 0, page_size: 20)
         booking_ids = data.map { |o| o.fetch('bookingId') }
         sentence_details = HmppsApi::PrisonApi::OffenderApi.get_bulk_sentence_details(booking_ids)
 
+        complexities = HmppsApi::ComplexityApi.get_complexities offender_nos
```

Comment:
Any chance we could apply the same `womens_prison?` logic here, to avoid calling out when listing offenders in a male prison?

I suppose the call to `PrisonService::womens_prison?(prison_code)` could even be split out into a private helper method within this class – e.g. `get_complexity?` – which would better explain what the intention is of this conditional logic.

Reply:
```suggestion
        complexities = if PrisonService::womens_prison?(prison)
                         HmppsApi::ComplexityApi.get_complexities(offender_nos)
                       else
                         {}
                       end
```